modules/core/src/index.ts

```diff
@@ -1,3 +1,4 @@
+import './lib/init';
```

Comment:
This file wasn't used?

Reply:
It was used before. We using it for loaders global version injection.